docs/ai/conceptual/prompt-engineering-dotnet.md

```diff
@@ -122,81 +113,63 @@ DOMESTIC POLICY
-
-By deeply integrating with Visual Studio Code, Semantic Kernel also makes it easy for you to integrate [prompt engineering](/semantic-kernel/prompts/your-first-prompt?tabs=Csharp) into your existing development processes:
-
-- Create prompts directly in your preferred code editor.
-- Write tests for your prompts using your existing testing frameworks.
-- Deploy your prompts to production using your existing CI/CD pipelines.
-
-Semantic Kernel Completions use GPT-3 and GPT-3.5 models, which have no specific format rules for prompts. The preceding sections all contain examples of Semantic Kernel Completions.
-
-Semantic Kernel Chat Completions use GPT-35-Turbo and GPT-4 models, which use a specific chat-like format consisting of role-based messages as shown in the following example.
-
-```csharp
-prompt = """
-<message role=""system"">
-    You are helping students with US History homework. Answers should not be about other countries except in relation to the US. Provide some supporting information for answers. Format any summaries using lists.
-</message>
-<message role=""user"">
-    Which president had the shortest term?
-<message/>
-<message role=""assistant"">
-    William Henry Harrison died just 31 days after his inauguration as president in 1841, making his presidency the shortest in U.S. history.
-<message/>
-<message role=""user"">
-    Instructions: Summarize US Presidential accomplishments, grouped by category. Categories: Domestic Policy, US Economy, Foreign Affairs, Space Exploration, Other.
-</message>
-""";
-```
-
-### Azure OpenAI
-
-[Azure OpenAI](/azure/ai-services/openai/overview) is a managed service that allows developers to deploy, tune, and generate content from OpenAI models on Azure resources. Azure OpenAI's client library for .NET is an adaptation of OpenAI's REST APIs that provides an idiomatic interface and rich integration with the rest of the Azure SDK ecosystem. It can connect to Azure OpenAI resources or to the non-Azure OpenAI inference endpoint, making it suitable for non-Azure OpenAI development.
+## Example prompt using .NET
 
-Use the client library for Azure OpenAI to:
+.NET provides various tools to prompt and chat with different AI models. Use [Semantic Kernel](../semantic-kernel-dotnet-overview.md) to connect to a wide variety of AI models and services, as well as other SDKs such as the official [OpenAI .NET library](https://www.nuget.org/packages/OpenAI-DotNet/). Semantic Kernel includes tools to author prompts with different roles and maintain chat history, as well as many other features.
```

Comment:
I think the last sentence is missing a word after "tools to"? It doesn't quite make sense to me.